src/csharp/Microsoft.Spark.Experimental/Sql/UdfRegistrationExtensions.DataFrame.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Data.Analysis;
+using Microsoft.Spark.Utils;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Extension methods for <see cref="UdfRegistration"/>.
+    /// </summary>
+    public static class DataFrameUdfRegistrationExtensions
```

Comment:
The type name and the file name should match. The file should be named `DataFrameUdfRegistrationExtensions.cs`.